web/src/components/map/menu.js

```diff
@@ -88,6 +89,11 @@ export const MapMenu = ({
           <List.Item>None</List.Item>
         )}
       </List>
+      {!collectionsDisplay.length && 
```

Comment:
`collectionsDisplay` excludes the collection actively being mapped. It displays a list of the *other* collections from that day. Consequently, it could be empty when there is still a collection being mapped. 

`collectionsOnDate` provides all of the collections and indicates whether we should display a warning message.

```suggestion
      {!collectionsOnDate.length && 
```

Reply:
Thanks Tim, I've pushed up my changes.